### DISCUSSION TITLE
Gnome nautilus context menu black border

### DISCUSSION THREAD

**Discussion by leandergangso** [i3/i3#5747] [Q&A]
Hi, I'm getting this black border around the content menu (right-click) that I'm not able to remove.
Anybody know of a fix for this? (This only happens when using i3)

![image](https://github.com/i3/i3/assets/60340480/2207aba0-bea8-4b89-8b39-b8dec8ecbb75)

**Comment by themukeshmahiya** [comment]
same problem in my system started using thunar

**Comment by MalyugaSensei** [comment]
I managed to find a workaround for myself. I used picom with the property `shadow = false;`. If you need shadows everywhere except context menus, then you can set the property `menu`:
```
shadow = true;
wintypes: {
  menu = { shadow = false; }
}
```

**Comment by orestisfl** [comment]
> This only happens when using i3

It doesn't only happen with i3, it happens with any standalone window manager without compositing. See https://gitlab.gnome.org/GNOME/gtk/-/issues/4815

**Comment by Certainty1396** [comment]
I have the same problem, and the options on the context menu can not be selected easily, lagging
